Dockerfile.dev

```diff
@@ -0,0 +1,35 @@
+FROM node:13.10.1-slim
```

Comment:
Why use not use a LTS version? 14.15.4 is the latest LTS.
https://nodejs.org/en/download/releases/